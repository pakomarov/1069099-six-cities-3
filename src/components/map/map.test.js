import React from 'react';
import renderer from 'react-test-renderer';
import Map from './map.jsx';


it(`Should match snapshot of Map`, () => {
  const tree = renderer
    .create(<Map
      area={[1, 2]}
      markers={[[1, 2]]}
    />, {
      createNodeMock: () => {
        return document.createElement(`div`);
      }
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
