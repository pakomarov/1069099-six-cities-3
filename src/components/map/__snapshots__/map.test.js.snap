// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match snapshot of Map 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
/>
`;
